tests/test_config.py

```diff
@@ -205,6 +205,21 @@ def test_search_parents(self, tmpdir):
             old.chdir()
         assert config.toxinipath == toxinipath
 
+    def test_explicit_config_path(self, tmpdir):
```

Comment:
I actually tried that at some point locally but didn't find a nice enough way, as we need line 212 and 213 in both runs anyway. Do you have any suggestions?

That said - I'm happy to try that again this evening.